libr/core/cmd_api.c

```diff
@@ -236,11 +308,57 @@ R_API int r_cmd_call(RCmd *cmd, const char *input) {
 	return ret;
 }
 
+static int cmdstatus2int(RCmdStatus s) {
+	switch (s) {
+	case R_CMD_STATUS_OK:
+		return 0;
+	case R_CMD_STATUS_INVALID:
+		return -1;
+	case R_CMD_STATUS_EXIT:
+	default:
+		return -2;
+	}
+}
+
 R_API int r_cmd_call_parsed_args(RCmd *cmd, RCmdParsedArgs *args) {
```

Comment:
I actually intend to transform this into:
```C
R_API RCmdStatus r_cmd_call_parsed_args(....)
```
as it makes it clearer what's the return value. I think right now the `int` returned is used in unconsistent ways.

Reply:
I have not done this as it would require many changes in cmd.c as well and I don't want to add too many changes in this PR. Going to do a separate pR after this.